src/page/sign-in-step.page.ts

```diff
@@ -6,9 +6,9 @@ export class SignInStepPage {
   private loginSubmitButton: ElementFinder;
 
   constructor () {
-    this.emailField = $('#email');
-    this.passwdField = $('#passwd');
-    this.loginSubmitButton = $('#SubmitLogin > span');
+    this.emailField = $('#login_form #email');
+    this.passwdField = $('#login_form #passwd');
+    this.loginSubmitButton = $('#login_form #SubmitLogin');
```

Comment:
we don't need to change the email and password locators. And about the submit button, you can simplify it :)

Reply:
Done